pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "humanloop"
-version = "0.8.6b3"
```

Comment:
@harry-humanloop was this intentional?

Reply:
not really; this was from the first commit in this PR, by fern-bot, which i suspect comes from `fern generate`

(https://github.com/humanloop/humanloop-python/pull/21/commits/77b76f7dcd91cec808fdb6b7c4ae59cc3eac7799)